nixos/modules/i18n/input-method/ibus.nix

```diff
@@ -40,7 +35,7 @@ in
   options = {
     i18n.inputMethod.ibus = {
       engines = lib.mkOption {
```

Comment:
(Let's start a new thread to discuss this alternative method.)

Instead of checking nothing, we can use a more popular method to check that packages are from a specific package set.  This way we can both deleting `meta.isIbusEngine` and checking something.

```nix
      engines = lib.mkOption {
        type = lib.types.functionTo (lib.types.listOf lib.types.package);
        default = ibus-engines: [ ];
        defaultText = lib.literalExpression "ibus-engines: [ ]";
        example = lib.literalExpression "ibus-engines: [ ibus-engines.mozc ibus-engines.hangul ]";
      }
```